bin/pycbc_inspiral

```diff
@@ -72,6 +72,9 @@ parser.add_argument("--max-template-length", type=float,
                   help="The maximum length of a template is seconds. The "
                        "starting frequency of the template is modified to "
                        "ensure the proper length")
+parser.add_argument("--q-transform", type=bool, default=False,
```

Comment:
Make option --enable-q-transform. Use action='store_true', then remove the type and default.